src/ngx_http_upsync_module.c

```diff
@@ -2391,14 +2395,13 @@ ngx_http_upsync_add_timers(ngx_cycle_t *cycle)
     if (upsync_server == NULL) {
         return NGX_OK;
     }
-    upscf = upsync_server->upscf;
 
     ngx_log_debug1(NGX_LOG_DEBUG_HTTP, cycle->log, 0,
-                   "upsync_add_timers: shm_name: %V", upsync_server->pc.name);
+        "upsync_add_timers: shm_name: %V", "upsync_add_timers");
```

Comment:
"upsync_add_timers" that is the type of str, not the type of ngx_str_t, cannt use "%V".

Just be replaced by ' ngx_log_debug0(NGX_LOG_DEBUG_HTTP, cycle->log, 0, "upsync_add_timers"); ' in one line will be ok. thanks.